template/module/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- encoding: utf-8 -*-
```

Comment:
That's one of the reasons I need this in the template. I always confuse those two.